ownable.sol

```diff
@@ -25,6 +25,12 @@ contract Ownable {
     address public owner;
     address public contentSpace;
 
+    // TODO: don't know if we can easily change this to msg.sender...
+    //  since object in the content hierarchy are created through factories, msg.sender would end up being the factory
+    //  instead of the end-user as expected. it theory we could change the space to set the owner as msg.sender but
```

Comment:
We can deal with that by adding in the factory a check that (msg.sender == tx.origin) that would avoid having the creation hijacked by another object above

Reply:
I think that check would need to be in the space to be effective.

Reply:
Made changes in space.